src/renderers/shaders/ShaderChunk/dithering_pars_fragment.glsl

```diff
@@ -0,0 +1,14 @@
+// based on https://www.shadertoy.com/view/MslGR8
+vec3 dithering( vec3 color ) {
+	//Calculate grid position
+	float grid_position = fract( dot( gl_FragCoord.xy - vec2( 0.5, 0.5 ) , vec2( 1.0 / 16.0, 10.0 / 36.0 ) + 0.25 ) );
+
+	//Shift the individual colors differently, thus making it even harder to see the dithering pattern
+	vec3 dither_shift_RGB = vec3( 0.25 / 255.0, -0.25 / 255.0, 0.25 / 255.0 );
+
+	//modify shift acording to grid position.
+	dither_shift_RGB = mix( 2.0 * dither_shift_RGB, -2.0 * dither_shift_RGB, grid_position );
+
+	//shift the color by dither_shift
+	return color + 0.5 / 255.0 + dither_shift_RGB;
```

Comment:
Hm.... here is a some discussion of the OpenGL specification on this matter:

https://www.opengl.org/discussion_boards/showthread.php/190960-OpenGL-float-normalized-integer-truncation-(need-to-add-0-5-255-to-outgoing-data-)

> https://www.opengl.org/registry/doc/glspec45.core.pdf
> section 2.3.5.2
> "...The conversion from a floating-point value f to the corresponding unsigned normalized fixed-point value c is defined by first clamping f to the range [0, 1], then computing f'=convert_float_uint(f*(2^b−1),b) where convert_float_uint(r, b) returns one of the two unsigned binary integer values with exactly b bits which are closest to the floating-point value r (where rounding to nearest is preferred)..."

This suggests that OpenGL 4.5 recommends rounding, but it is only a recommendation.  My thoughts are that we should not be adding a half bit.